Formula/openjdk.rb

```diff
@@ -46,7 +46,8 @@ def install
     ENV["MAKEFLAGS"] = "JOBS=#{ENV.make_jobs}"
     system "make", "images"
 
-    libexec.install "build/macosx-x86_64-server-release/images/jdk-bundle/jdk-#{short_version}.jdk" => "openjdk.jdk"
+    jdk = Dir["build/*/images/jdk-bundle/*"].first
+    libexec.install jdk => "openjdk.jdk"
```

Comment:
this is needed because `short_version` maps to `14.0.0`but it gets built as `jdk-14.jdk`